src/Compilers/Core/Portable/SpecialType.cs

```diff
@@ -255,9 +255,14 @@ public enum SpecialType : sbyte
         /// </summary>
         System_Runtime_CompilerServices_RuntimeFeature = 44,
 
+        /// <summary>
+        /// An attribute that is placed on each method with a 'methodimpl" aka ".override" in metadata.
```

Comment:
>An attribute that is placed on each method with a 'methodimpl" aka ".override" in metadata. [](start = 12, length = 91)

By C# compiler? VB doesn't do that, correct?

Reply:
VB doesn't current do that, but it could and should.